Rakefile

```
@@ -61,6 +61,10 @@ def env_var(name, default=nil)
   value
 end
 
+def dry_run?
+  env_var("DRY_RUN", default="true") == "true"
```

Why do we need `default=` here? It defines and assigns a local variable `default`.

```suggestion
  env_var("DRY_RUN", "true") == "true"
```

---

BTW, is it expected that we use dry run mode by default. In general, dry run mode is disabled by default. For example, `git`, `rsync`, ...

---

fix: [f74a654](https://github.com/groonga/groonga/pull/2324/commits/f74a654f72d6b52469cca1d697813066678464b0) I understood it. (I was afraid that I accidentally post announces on X during implementations. )